admin/cli/cli.go

```diff
@@ -63,9 +83,7 @@ type CmdWithContextRunner interface {
 	RunCmdWithContext(context.Context, *flags.GlobalFlags) (commands.Result, error)
 }
 
-// Run function is a top-level function which handles running all commands
-// in a standard way based on the interface they implement.
-func (c *Commands) Run(ctx *kong.Context, globals *flags.GlobalFlags) error {
+func runGeneric(ctx *kong.Context, globals *flags.GlobalFlags) error {
```

Comment:
nitpick: we can simply call this function `run()`

Reply:
great catch